downstream/modules/eda/proc-eda-set-up-new-project.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc-eda-set-up-new-project"]
+
+= Setting up a new project
+
+[role="_abstract"]
+
+Projects are a logical collection of rulebooks.
+
+== Prerequisites
+
+* You are logged in to the {EDAcontroller} Dashboard as a Content Consumer.
+* You have set up a credential, if necessary. 
+For more information, refer to the link:https://docs.ansible.com/automation-controller/latest/html/userguide/credentials.html[Credentials]
+section of the {ControllerNameStart} documentation.
+* You have an existing repository containing rulebooks that are integrated with playbooks contained in a repository to be used by Automation Controller.
+
+== Procedure
+
+. Log in to the {EDAcontroller} Dashboard.
+. From the navigation bar, select btn:[Projects] → btn:[Create project].
+. Insert the following:
+** *Name*
+** *Description*: This field is optional.
+** *SCM type*: Git is the only SCM type available for use.
+** *SCM URL*: This is a URL to a remote archive, such as GitHub Release or a build artifact stored in Artifactory that unpacks it into the project path for use.
++
+[NOTE]
+====
+You cannot edit the SCM URL after you create the project.
+====
+** *Credential*: This is the token needed to utilize the SCM URL.
+. Select btn:[Create project].
+
+Your project is now created and can be managed in the *Projects* screen.
+
+After saving the new project, you are brought to the project's details page. From there or the *Projects* list view you can edit or delete the project.
```

Comment:
Suggest changing to:

After saving the new project, the project details page is displayed.

Also suggest adding a comma:
From there or the Projects list view, you ...